components/private_channel/browser/private_channel_service.cc

```diff
@@ -0,0 +1,320 @@
+/* Copyright 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <utility>
+
+#include "brave/components/private_channel/browser/private_channel_service.h"
+#include "brave/components/private_channel/browser/static_values.h"
+#include "brave/components/private_channel/browser/request_utils.h"
+#include "brave/components/private_channel/client_private_channel.h"
+#include "brave/components/private_channel/utils.h"
+
+#include "base/logging.h"
+#include "extensions/common/url_pattern.h"
```

Comment:
you're missing a dep for this, but I don't see where it's used anyway

Reply:
- [x] removed `"extensions/common/url_pattern.h"` include